sprints/dashboard/utils.py

```diff
@@ -605,3 +606,20 @@ def get_sprint_meeting_day_division(sprint_start: str) -> DefaultDict[str, Tuple
             break
 
     return result
+
+
+def create_mattermost_post(message: str, email_ids: List[str]) -> None:
```

Comment:
I think it might make sense to move this to `libs.mattermost`. I took a second look at this file and the number of lines grew tremendously here since I last checked it.

Reply:
I agree, also files often like `common` and `utils` tends to become a dumping ground for functions, but maybe that refactoring is a part of another day. ;)

Reply:
Yep, let's leave the general refactor for another time, but we can avoid creating more functions here.